README.md

```diff
@@ -1,5 +1,23 @@
 ## Woopra Android SDK
 
+## Installation
+To declare the mavenCentral() repository. Add mavenCentral() in the project-level build.gradle or
+in the app module's build.gradle.
```

Comment:
imo: add blank lines before and after code blocks to improve readability and also potentially conform to the markdownlint: https://github.com/markdownlint/markdownlint